index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project</title>
+    <meta name="viewport" content="width=device-width">
+    <link href="style.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+<header class="main">
+    <h1>Reliable, efficient delivery</h1>
+    <h1><strong>Powered by Technology</strong></h1>
+    <p>Our Artificial Intelligence powered tools use millions of project data <br> points to ensure that your project is
+        successful </p>
+</header>
+<main class="container">
```

Comment:
It confused me that your <header> section has class name "main" while your <main> section has class name "container". Might it be clearer for your <header> to have a different class name to avoid repetition of the word 'main'?

Reply:
Ohh... Thank you I'll make corrections to that.